code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -135,16 +136,16 @@
 	parry_imperfect_falloff_percent = 20
 	parry_efficiency_perfect = 100
 	parry_efficiency_perfect_override = list(
-		ATTACK_TYPE_PROJECTILE_TEXT = 60,
+		TEXT_ATTACK_TYPE_PROJECTILE = 60,
 	)
 
 	parry_efficiency_considered_successful = 0.01
 	parry_efficiency_to_counterattack = 0.01
 	parry_max_attacks = INFINITY
-	parry_failed_cooldown_duration =  1.5 SECONDS
-	parry_failed_stagger_duration = 0
-	parry_cooldown = 0
-	parry_failed_clickcd_duration = 0.8
+	parry_failed_cooldown_duration =  3 SECONDS
+	parry_failed_stagger_duration = 2 SECONDS
```

Comment:
These values feel as though they're mixed up with standard unarmed, as above. These values make pugilist parrying fairly unwieldy compared to standard unarmed parrying.